vscode-extension/src/extension.ts

```diff
@@ -228,54 +226,29 @@ async function createNewAIConfig(
   );
 
   const fileContentPath = mode === "json" ? newAIConfigJSON : newAIConfigYAML;
-
   const fileContentBuffer = await vscode.workspace.fs.readFile(fileContentPath);
   const initialContent = fileContentBuffer.toString();
 
   const workspacePath = vscode.workspace.workspaceFolders
     ? vscode.workspace.workspaceFolders[0].uri.path
     : null;
 
-  // Find the first available untitled file name to suggest as default
-  let firstAvailableUntitledName: string | null = null;
-  let i = 0;
-  while (firstAvailableUntitledName === null) {
-    const fileName = `untitled${i === 0 ? "" : "-" + i}.aiconfig.${mode}`;
-    const filePath = workspacePath
-      ? path.join(workspacePath, fileName)
-      : fileName;
-
-    if (fs.existsSync(filePath)) {
-      i++;
-    } else {
-      firstAvailableUntitledName = fileName;
-    }
-  }
-
-  const newConfigName = await vscode.window.showInputBox({
-    prompt: "Enter a name for the new AIConfig file",
-    value: firstAvailableUntitledName,
-    validateInput: (input) => validateNewConfigName(input, mode),
-  });
-
+  const untitledFileName = `untitled.aiconfig.${mode}`;
```

Comment:
I like having the `firstAvailableUntitledName` like what we had before, but get now that we're not actually saving it yet, so maybe there's a way to check for both existing filenames that exist, and also files that are currently open. Not a blocker